academic-services/knowledge-exploration-service/reference-get-evaluate.md

```diff
@@ -0,0 +1,411 @@
+---
+title: GET Evaluate
+description: Returns entities that match constraints specified in a structured query expression
+ms.topic: reference
+ms.date: 2020-02-07
+---
+
+# Evaluate REST API
+
+The **Evaluate** API returns entities that match constraints specified in a [structured query expression](concepts-query-expressions.md).
+
+Query expressions can either be manually generated using the [structured query expression](concepts-query-expressions.md) syntax and [entity schema](reference-makes-api-entity-schema.md), or automatically generated by interpreting a natural language query using the [Interpret method](reference-get-interpret.md).
+
+``` HTTP
+GET http://{serviceName}.{serviceRegion}.cloudapp.azure.com/evaluate?expression={expression}
+```  
+
+With optional parameters:
+
+``` HTTP
+GET http://{serviceName}.{serviceRegion}.cloudapp.azure.com/evaluate?expr={expr}&attributes={attributes}&offset={offset}&count={count}&orderby={orderby}
+```  
+
+## URI Parameters
+
+Name | Required | Type | Description
+--- | --- | --- | ---
+`expr` | Required | string | Structured query expression to evaluate. See [structured query expressions](concepts-query-expressions.md) for documentation.
+`orderby` | Optional | string | The attribute to use for sorting the result set, followed by an optional sort order indicator in the format "asc" for ascending and "desc" for descending. E.g. Year:desc would return entities ordered newest first. <br/><br/>Note that changing the default order of large result sets can take a very long time, so changing sort order should be used cautiously. <br/><br/>The default value is entity log probability score in descending order.
+`attributes` | Optional | string | A list of comma-separated attributes to include for each entity in the result set. See the [entity schema](reference-makes-api-entity-schema.md) for the attributes that can be requested. Returned entities aren't required to have the attributes, which means the list can include attributes from different entity types. This is useful when the query expression potentially matches multiple types of entities. <br/><br/>If no attributes are specified then the each entity will only contain its corresponding score. <br/><br/>If an asterisk (*) is specified, all available attributes will be returned. <br/><br/>Defaults to an empty string.
```

Comment:
>Returned entities aren't required to have the attributes [](start = 222, length = 56)

This took me two times to read.  Maybe this?
"Attributes in the list do not need to be present on every entity returned.  You may include attributes from different entity types if you like."